content/docs/instrumenting/clientlibs.md

```diff
@@ -24,6 +24,7 @@ Unofficial third-party client libraries:
 * [Haskell](https://github.com/fimad/prometheus-haskell)
 * [Node.js](https://github.com/siimon/prom-client)
 * [.NET / C#](https://github.com/andrasm/prometheus-net)
+* [Lua](https://github.com/knyar/nginx-lua-prometheus) for Nginx
```

Comment:
Moved. Also swapped `Node.js` and `.NET / C#`, since otherwise it's not immediately obvious that the list should be sorted alphabetically.